packages/clerk-js/src/ui/polishedAppearance.ts

```diff
@@ -156,21 +156,27 @@ export const polishedAppearance: Appearance = {
         borderWidth: 0,
         boxShadow: `${BORDER_SHADOW_LENGTH} ${theme.colors.$neutralAlpha150}, ${theme.shadows.$badge}`,
       },
-      phoneInputBox: (() => {
-        const boxShadow = [
-          `0px 0px 0px 1px ${theme.colors.$neutralAlpha150}`,
-          theme.shadows.$input.replace('{{color}}', theme.colors.$neutralAlpha100),
-        ].toString();
-        return {
-          borderWidth: 0,
-          '> div > input': {
+      phoneInputBox: {
+        ...inputStyles(theme),
+        'div > input': {
+          boxShadow: 'unset',
+          '&[data-feedback]': {
+            boxShadow: 'unset',
+            '&:focus-within': {
```

Comment:
Why do we need to unset all these since the phone input has the same behaviour with simple styles on?

Reply:
The first one is mandatory as it adds a shadow to the input that don't match the designs.
The `boxShadow: 'unset'` under `&[data-feedback]` can be removed

Reply:
We are applying shadows for every input here https://github.com/clerk/javascript/pull/2849/files/dab67cc399f42811f9cde6ebdcf787d317c3d09b#diff-06c7402c7f9eeed7be187342030203f1b5279e81bce2cede4c26421430f619d9L134